inline_test.go

```diff
@@ -973,6 +973,28 @@ func TestFootnotesWithParameters(t *testing.T) {
 	doTestsInlineParam(t, tests, Options{Extensions: EXTENSION_FOOTNOTES}, HTML_FOOTNOTE_RETURN_LINKS, params)
 }
 
+func TestInlineComments(t *testing.T) {
+	var tests = []string{
+		"Hello <!-- there ->\n",
+		"<p>Hello &lt;!&mdash; there &ndash;&gt;</p>\n",
+		"Hello <!-- there -->\n",
+		"<p>Hello <!-- there --></p>\n",
+		"Hello <!-- there -->",
+		"<p>Hello <!-- there --></p>\n",
+		"Hello <!---->\n",
+		"<p>Hello <!----></p>\n",
+		"Hello <!-- there -->\na",
+		"<p>Hello <!-- there -->\na</p>\n",
+		"* list <!-- item -->\n",
+		"<ul>\n<li>list <!-- item --></li>\n</ul>\n",
+		"<!-- Front --> comment\n",
+		"<p><!-- Front --> comment</p>\n",
+		"blahblah\n<!--- foo -->\nrhubarb\n",
+		"<p>blahblah\n<!--- foo -->\nrhubarb</p>\n",
```

Comment:
Maybe it's a good idea to separate each test case (input and expected output) with a blank line, for better readability and consistency with existing code. See [here](https://github.com/russross/blackfriday/blob/1d673b3cd9cc47e45ced0b25a9aa191dbf8cfbd8/inline_test.go#L162-L181) for example.